javalin/src/main/java/io/javalin/plugin/rendering/vue/JavalinVue.kt

```diff
@@ -103,3 +103,14 @@ object PathMaster {
 }
 
 fun Path.readText() = String(Files.readAllBytes(this))
+
+fun escape(string: String?) = string?.toCharArray()?.map {
```

Comment:
The intention is to use the `escape()` method rather than JDK `URLEncoder.encode()` ?

Reply:
Yes, the params are already url-decoded, but need to be HTML escaped, since they're put in what is basically a template file.